roles/acm_setup/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
         disableHubSelfManagement: "{{ hub_disable_selfmanagement }}"
 
 - name: "Wait for search-postgres deployment"
-  k8s_info:
+  community.kubernetes.k8s_info:
```

Comment:
I think is better to get this fixed in a different change to avoid the explosion of issues that could prevent others to merge changes

Reply:
After our sync, we agreed to include this one as upcoming changes will compare to this, so we shouldn't see much issues, despite being there :+1: